mreg_cli/api/history.py

```diff
@@ -5,36 +5,47 @@
 import datetime
 import json
 from enum import Enum
-from typing import Any
+from typing import Any, Self
 
 from pydantic import BaseModel, Field, field_validator
 
 from mreg_cli.api.endpoints import Endpoint
 from mreg_cli.exceptions import EntityNotFound, InternalError
 from mreg_cli.outputmanager import OutputManager
-from mreg_cli.utilities.api import get_list
+from mreg_cli.utilities.api import get_typed
 
 
 class HistoryResource(str, Enum):
```

Comment:
Discovered that the changes to the `HistoryResource` enum that was proposed in #232 but then merged in #237 because of some rebasing issues, got lost when trying to fix the branch. It has been added again here along with the refactored `HistoryItem.get()` method.